pcdscalc/be_lens_calcs.py

```diff
@@ -181,37 +180,54 @@ def get_lens_set(set_number_top_to_bot, filename=None):
         before experiments, this is to specify what number set.
     filename : str, optional
         File path of the lens_set file.
-        This is a binary file generated with `numpy.save` which saves an array
-        to a binary file in NumPy .npy format.
+    get_all : bool
+        Indicates if it should return the entire stack of lens.
 
     Returns
     -------
     lens_set : list
         [numer1, lensthick1, number2, lensthick2 ...]
     """
     if filename is None and LENS_SET_FILE is None:
-        logger.error('You must provide the path to the lens_set file or you '
-                     'must configure it via :meth: `configure_lens_set_file`')
-        return
+        err_msg = ('You must provide the path to the lens_set file or you '
+                   'must configure it via configure_lens_set_file function')
+        logger.error(err_msg)
+        raise ValueError(err_msg)
     elif filename is None:
         filename = LENS_SET_FILE
     if not os.path.exists(filename):
-        logger.error('Provided invalid path for lens set file: %s', filename)
-        return
+        err_msg = ('Provided invalid path for lens set file: %s', filename)
+        logger.error(err_msg)
+        raise FileNotFoundError(err_msg)
     if os.stat(filename).st_size == 0:
-        logger.error('The file is empyt: %s', filename)
-        return
-    with open(filename, 'rb') as lens_file:
-        sets = np.load(lens_file, allow_pickle=True)
-        if set_number_top_to_bot not in range(1, sets.shape[0]):
-            logger.error('Provided and invalid set_number_top_to_bottom %s, '
-                         'please provide a number from 1 to %s ',
-                         set_number_top_to_bot, sets.shape[0])
-            return
-    return sets[set_number_top_to_bot - 1]
-
-
-def set_lens_set_to_file(sets_list_of_tuples, filename,
+        err_msg = ('The file is empyt: %s, use set_lens_set_to_file to write '
+                   'set lenses to the file.', filename)
+        logger.error(err_msg)
+        raise ValueError(err_msg)
```

Comment:
This one has a typo (empyt) as well as being a tuple

Reply:
thank you, fixed it in ef27fed